pipelines/coref/configs/cluster.cfg

```diff
@@ -0,0 +1,135 @@
+[paths]
+train = "data/train.spacy"
+dev = "data/dev.spacy"
+init_tok2vec = null
+vectors = null
+
+[system]
+gpu_allocator = "pytorch"
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["sentencizer", "transformer", "coref"]
```

Comment:
I don't think we need the `sentencizer` for this component only for `span-resolver`.

Reply:
That's correct, I'll remove it.